src/assets/sass/protocol/components/_navigation.scss

```diff
@@ -50,6 +50,34 @@
     }
 }
 
+/* -------------------------------------------------------------------------- */
+// Sticky navigation styles
+
+@supports (position: sticky) {
+    html.mzp-has-sticky-navigation {
+        .mzp-c-navigation.mzp-is-sticky {
+            @include transition(transform 300ms ease-in-out);
+            left: 0;
+            position: sticky;
+            top: 0;
+            z-index: 1000;
+
+            &.mzp-is-scrolling {
+                box-shadow: 0 0 5px 1px rgba(9, 32, 77, .12);
+            }
+
+            &.mzp-is-hidden {
+                @include transform(translate(0, -100%));
```

Comment:
The shadow is still visible since it extends past the height of the nav, so we should make this a bit more than 100% to compensate. Maybe just `-101%` or we could even do `calc(-100% - 8px)`

Reply:
Seems like `-105%` seems to be the right amount. I went with a static value, as opposed to throwing `calc()` insto the mix.